scripts/automate_release.py

```diff
@@ -0,0 +1,320 @@
+import json
+import os
+
+
+versions = os.environ['VERSIONS']
+linux_matricies = os.environ['LINUX_MATRICIES']
```

Comment:
To populate these matrixes we need to call workflow : pytorch/test-infra/.github/workflows/generate_binary_build_matrix.yml@main
These values can't bee collected from the environment variables.